From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Sam Thursfield <sam.thursfield@codethink.co.uk>
Date: Mon, 17 Oct 2022 11:32:00 +0200
Subject: [PATCH] backport: seccomp: Allow epoll_create1()

This is preferred over epoll_create(), and is used in GLib 2.74 and
later due to https://gitlab.gnome.org/GNOME/glib/-/merge_requests/2868

https://gitlab.gnome.org/GNOME/tracker-miners/-/issues/239
---
 src/libtracker-miners-common/tracker-seccomp.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/libtracker-miners-common/tracker-seccomp.c b/src/libtracker-miners-common/tracker-seccomp.c
index 334daab0787510a6179cb47f3914dc2833244f66..ec271495fe986b6b4999be2a10321c45c1588133 100644
--- a/src/libtracker-miners-common/tracker-seccomp.c
+++ b/src/libtracker-miners-common/tracker-seccomp.c
@@ -208,6 +208,7 @@ tracker_seccomp_init (void)
 	ALLOW_RULE (pipe);
 	ALLOW_RULE (pipe2);
 	ALLOW_RULE (epoll_create);
+	ALLOW_RULE (epoll_create1);
 	ALLOW_RULE (epoll_ctl);
 	/* System */
 	ALLOW_RULE (uname);
